.github/workflows/build_lint.yml

```diff
@@ -60,18 +60,24 @@ jobs:
       with:
         path: ${{ env.pythonLocation }}
         key: ${{ runner.os }}-pip-${{ env.pythonLocation }}-${{ hashFiles('setup.py') }}
+    # Workaround; remove once https://github.com/astral-sh/uv/issues/1386 is fixed
+    - run: echo "VIRTUAL_ENV=${Python_ROOT_DIR}" >> $GITHUB_ENV
+    - name: Install uv
+      run: pip install uv
     - name: Install dependencies
       # Only if the cache misses
       # Based on https://github.com/pypa/pip/issues/8049#issuecomment-633845028
       # read_requirements.py should be removed once
       # https://github.com/pypa/pip/issues/11440 is resolved.
       if: steps.cache.outputs.cache-hit != 'true'
       run: |
-        pip install toml
+        rm -fr ${Python_ROOT_DIR}/lib/python${{ matrix.python-version}}/site-packages/pip-23.0.1.dist-info
+        rm -fr ${Python_ROOT_DIR}/lib/python${{ matrix.python-version}}/site-packages/pip-23.3.1.dist-info
+        uv pip install toml
         python tests/read_requirements.py > requirements.txt
-        pip install -r requirements.txt
+        uv pip install -r requirements.txt
     - name: Install Mesa
-      run: pip install --no-deps .
+      run: uv pip install --no-deps "mesa @ ."
```

Comment:
Is the "mesa @" part required? Locally it works just fine without that

Reply:
Used to require such. It could have been fixed by 0.1.5. Let me try it.

Reply:
Still requires "mesa @ .".